src/components/OraclesPanel.vue

```diff
@@ -25,9 +25,6 @@ import { storeToRefs } from 'pinia'
 import { computed, ref } from 'vue'
 import { useRoute, useRouter } from 'nuxt/app'
 import { useOraclesStore } from '@/stores/oracles'
-import OraclesTableCondensed from '@/components/OraclesTableCondensed'
```

Comment:
Apparently after the upgrade the imports that can be auto-imported are removed by linter. Which is a good thing